games/game_bladeandsorcery.py

```diff
@@ -1,18 +1,198 @@
+from pathlib import Path
+import json
+import os
+from typing import List
+
+from PyQt6.QtCore import QDateTime, QDir, QLocale, Qt
+from PyQt6.QtGui import QFont
+
+import mobase
+
+from PyQt6.QtWidgets import QSizePolicy, QVBoxLayout, QFormLayout, QLabel, QStyle
+from ..basic_features.basic_save_game_info import (
+    BasicGameSaveGame,
+    BasicGameSaveGameInfo
+)
+
 from ..basic_game import BasicGame
 
 
-class BaSGame(BasicGame):
+class BaSSaveGame(BasicGameSaveGame):
+    def __init__(self, filepath):
+        super().__init__(filepath)
+        self._filepath = Path(filepath)
+        save = open(self._filepath, "rb")
+        save_data = json.load(save)
+        self._gameMode: str = save_data["gameModeId"]
+        self._gender: str = "Male" if save_data["creatureId"] == "PlayerDefaultMale" else "Female"
+        self._ethnicity: str = save_data["ethnicGroupId"]
+        h, m, s = save_data["playTime"].split(":")
+        self._elapsed: tuple[int, int, float] = (int(h), int(m), float(s))
+        self._created: float = os.path.getctime(filepath)
```

Comment:
I think this would be cleaner for modern Python:

```
f_stat = self._filepath.stat()
self._created = f_stat.st_ctime
self._modified = f_stat.st_mtime
```